dataclients/kubernetes/ingressv1.go

```diff
@@ -0,0 +1,429 @@
+package kubernetes
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/dataclients/kubernetes/definitions"
+	"github.com/zalando/skipper/eskip"
+)
+
+func setPathV1(m PathMode, r *eskip.Route, pathType, path string) {
```

Comment:
Should we check for empty path like in beta?
```go
	if p == "" {
		return
	}
```

Reply:
That's interesting if someone specifies a valid v1 pathType but no path.

Reply:
Empty paths are not path rules. 
Example source file:
```
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: qux
  namespace: foo
spec:
  rules:
  - host: www.example.org
    http:
      paths:
      - path: ""
        pathType: ImplementationSpecific
        backend:
          service:
            name: qux
            port:
              name: baz
```

Example runtime ingress fetched:
```
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: qux
  namespace: foo
  resourceVersion: "1022182097"
  uid: de7c7d39-c22b-496e-ac7f-41471319c4f5
spec:
  rules:
  - host: www.example.org
    http:
      paths:
      - backend:
          service:
            name: qux
            port:
              name: baz
        pathType: ImplementationSpecific
status:
  loadBalancer: {}
```

Reply:
checked the code you are right it would change behavior if you set pathType to `Prefix` or `Exact`, thanks again for the review!